pgoapi/player_stats.py

```diff
@@ -0,0 +1,30 @@
+
+class PlayerStats:
+    def __init__(self, player_stats):
+        self.player_stats = player_stats
+        self.experience = 0
+        self.next_level_xp = 0
+        self.prev_level_xp = 0
+        self.unique_pokedex_entries = 0
+        self.km_walked = 0
+        self.level = 0
+        self.parse_values()
+
+    def parse_values(self):
+        self.experience = self.player_stats.get('experience', 0)
+        self.next_level_xp = self.player_stats.get('next_level_xp', 0)
+        self.prev_level_xp = self.player_stats.get('prev_level_xp', 0)
+        self.unique_pokedex_entries = self.player_stats.get('unique_pokedex_entries', 0)
+        self.km_walked = self.player_stats.get('km_walked', 0)
+        self.level = self.player_stats.get('level', 0)
+
+    def __str__(self):
```

Comment:
can you add .encode('utf-8', 'ignore') after the format() some people had issues with char encoding on windows looks like this fixes it. same in the player.py